tournament/src/api/views/manage_tournament_views.py

```diff
@@ -73,6 +74,11 @@ def delete(request: HttpRequest, tournament_id: int) -> JsonResponse:
                 'error': f'you cannot delete `{tournament_name}` because you are not the owner of the tournament'
             }, status=403)
 
+        validation = request.GET.get('validation')
+        excepted_validation = f'{user["username"]}/{tournament_name}'
+        if validation != excepted_validation:
+            return JsonResponse({'error': error.INVALID_DELETE_VALIDATION}, status=400)
```

Comment:
I don't believe it's a good idea to add a passphrase requirement for deleting a tournament in your API. This is something that we would now need to implement on the front end of the site. If we decide to proceed with it, we can simply prompt the user to input a passphrase on the front end instead of within the API. I just don't think this is something that belongs in the API.